shell.nix

```diff
@@ -14,7 +14,7 @@ pkgs.mkShell {
       libevent
       zeromq
       sqlite
-      db48
+      db48 # Berkeley DB 4.8
```

Comment:
we might also want have a way to enable/disable this at some point

Reply:
Agreed, what do you think of my approach of enabling/disabling things in cf1e4393178222ee45a35d7a8c514225ab156034?

Reply:
TIL `nix-shell --arg`! I like it.

Reply:
Now covered in #7.